init.c

```diff
@@ -1370,6 +1370,11 @@ static enum CommandResult parse_mailboxes(struct Buffer *buf, struct Buffer *s,
     {
       //error
       mailbox_free(&m);
+      if (new_account)
+      {
+        TAILQ_REMOVE(&AllAccounts, TAILQ_LAST(&AllAccounts, AccountList), entries);
```

Comment:
Am I missing something, or:
```suggestion
        TAILQ_REMOVE(&AllAccounts, a, entries);
```

Reply:
heh yes of course